shared-libs/validation/src/validation.js

```diff
@@ -178,6 +178,30 @@ module.exports = {
       logger.error('isISOWeek validation failed: the number of week is greater than the maximum');
       return Promise.resolve(false);
     },
+
+    isAfter: (doc, validation) => {
+      const fields = [...validation.funcArgs];
+      const duration = _parseDuration(fields.pop());
+      const testDate = moment(doc[validation.field]);
+      const controlDate = moment().add(duration);
```

Comment:
I'm not so sure about this. There's no way that anyone can specifically influence when the document will be validated. Comparing a date to present might end up with a variety of very hard to explain bugs. 

When some configuration was wrong, it won't just be a question of "which config added the error", but also "when the code ran" (which is something that is NOT stored anywhere). 

Is there a way that we can get this functionality and compare with a fixed date on the report?

Reply:
How about comparing against the reported date?

Reply:
Comparing against reported date sounds great!